src/commands/sourcemaps/__tests__/utils.test.ts

```diff
@@ -12,14 +12,4 @@ describe('utils', () => {
       )
     })
   })
-  describe('buildPath', () => {
```

Comment:
now in https://github.com/DataDog/datadog-ci/pull/209/files#diff-a1cfa8cac5c81f1a10c4ff155546ac9b6beee67143d872c1c503a4c25f75c8a6R112-R121